doc/source/whatsnew/v0.15.0.rst

```diff
@@ -311,8 +311,7 @@ Timezone handling improvements
 - ``tz_localize(None)`` for tz-aware ``Timestamp`` and ``DatetimeIndex`` now removes timezone holding local time,
   previously this resulted in ``Exception`` or ``TypeError`` (:issue:`7812`)
 
-  .. ipython:: python
-     :okwarning:
+  .. code-block:: ipython
```

Comment:
if you change to a code-block you have to show the evaluated code

Reply:
fixed+green